packages/syntax-core/src/Checkbox/Checkbox.stories.tsx

```diff
@@ -49,3 +50,31 @@ const CheckboxInteractive = () => {
 export const Interactive: StoryObj<typeof Checkbox> = {
   render: () => <CheckboxInteractive />,
 };
+
+export const FixDoesNotBlowoutHeight: StoryObj<typeof Checkbox> = {
+  render: () => (
+    <Box height="100px" overflowY="auto">
```

Comment:
![image](https://github.com/Cambly/syntax/assets/127199/0c1c1bd4-cf20-431e-8608-fc313eec5488)
could we add some paddingX / Y to the container? Right now it breaks the focus styles

Reply:
To the container in the stories?   Is this unique to this story, or do all the focus styles brush up against the container like in this story?

I'll check.  if it is unique to this story, I'll add in some padding in the example.  If it is not unique to this one, then it'd be better I think to do that in a deparate PR for all of the stories. 👍

Reply:
@benjroy Yes, it's unique to this story because of the `overflow` 
![image](https://github.com/Cambly/syntax/assets/127199/9ddefd2f-bf9f-44b7-bd09-77f6e40dd056)